Content.Shared/_Starlight/Movement/Components/MovementBodyPartHinderedByClothing.cs

```diff
@@ -0,0 +1,12 @@
+using Robust.Shared.GameStates;
+using Robust.Shared.Physics;
+using Robust.Shared.Physics.Systems;
+
+namespace Content.Shared.Movement.Components;
+
+[RegisterComponent, NetworkedComponent]
+public sealed partial class MovementBodyPartHinderedByClothingComponent : Component
```

Comment:
if it's only about shoes clothing so rename or if it's not only about shoes, add ability to change "shoes" slot to another slot(s).

```suggestion
public sealed partial class MovementBodyPartHinderedByShoesComponent : Component
```

Reply:
Renamed the system to `MovementBodyPartHinderedByShoes` everywhere.